azure_arc_data_jumpstart/aks/arm_template/artifacts/DeployPostgreSQL.ps1

```diff
@@ -1,13 +1,15 @@
 Start-Transcript -Path C:\Temp\deployPostgreSQL.log
 
 # Deployment environment variables
-$controllerName = "Jumpstart-DC"
+$controllerName = "jumpstart-dc"
```

Comment:
This will also change the readme screenshot. need to update...

Reply:
@likamrat no problem. Let me submit a commit with the readme updates - had created a diagram to add onto the AKS scenario.

Reply:
Done